index.html

```diff
@@ -70,37 +70,38 @@ <h3 class="supporting-text">I’am a Software Developer</h3>
         <div class="social">
             <ul class="social-icons">
                 <li>
-                    <a href="#"><img src="./images/github.png" alt=""/></a>
+                    <a href="#"><img src="./images/github.png" alt="github icon"/></a>
                 </li>
 
                 <li>
-                    <a href="#"><img src="./images/Linkedin.png" alt=""/></a>
+                    <a href="#"><img src="./images/Linkedin.png" alt="Linkedin icon"/></a>
                 </li>
 
                 <li>
-                    <a href="#"><img src="./images/angelist.png" alt=""/></a>
+                    <a href="#"><img src="./images/angelist.png" alt="angelist icon"/></a>
                 </li>
 
                 <li>
-                    <a href="#"><img src="./images/twitter.png" alt=""/></a>
+                    <a href="#"><img src="./images/twitter.png" alt="twitter icon"/></a>
                 </li>
 
                 <li>
-                    <a href="#"><img src="./images/medium.png" alt=""/></a>
+                    <a href="#"><img src="./images/medium.png" alt="medium icon"/></a>
                 </li>
             </ul>
             </div>
         </div>
     </section>
 
     <section class="work-section">
+        <a id="work-section"></a>
         <div class="container">
         <div class="title">
             <h2 class="work-header">My Recent works</h2>
             <hr>
         </div>
         <div class="project-1">
-            <img class="w-image" src="./images/placeholder.png" alt=""/>
+            <img class="w-image" src="./images/placeholder.png" alt="first project image"/>
             <div class="right-block">
             <h2>Multi-Post Stories</h2>
```

Comment:
- [ ] To keep your HTML semantic, you should consider making this heading an `<h3>`. You already have a heading for this whole section, which is an `<h2>`, so hierarchically, this is lower. Also, all the other articles have an `<h3>` heading! You can read more about headings [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)